src/domain/challenge/hub/hub.service.ts

```diff
@@ -651,7 +651,7 @@ export class HubService {
 
   async createChallengeInHub(
     challengeData: CreateChallengeOnHubInput,
-    agentInfo?: AgentInfo
+    agentInfo: AgentInfo
```

Comment:
I am not sure why AgentInfo is optional here, but this change can potentially cause a regression.

Reply:
Reverted.